mage_ai/data_preparation/models/pipelines/integration_pipeline.py

```diff
@@ -212,6 +212,8 @@ def test_connection(self, block_type: BlockType, config: str = None):
                         error = dig(json_object, 'tags.error')
                     except Exception:
                         error = line
+                elif not error and line.startswith('CRITICAL'):
```

Comment:
Included this fallback cuz there may be instances where user clicks "Test connection" but the error is blank. For example, this scenario occurred when testing the "Test connection" for Commercetools and Stripe, but the fallback  error message has some details the user can act upon:

<img width="1386" alt="image" src="https://github.com/mage-ai/mage-ai/assets/78053898/af8e5b45-fe99-45a9-895e-388d08092c60">

<img width="1385" alt="image" src="https://github.com/mage-ai/mage-ai/assets/78053898/a62dada3-bf57-40f1-82e2-6d41f5acea87">